commands/__tests__/account.test.ts

```diff
@@ -0,0 +1,66 @@
+import yargs, { Argv } from 'yargs';
```

Comment:
This is mostly just a rename (from accounts.test.ts), but I also had to make some changes in here to make TS happy. The main difference is this:
```ts

const commandSpy = jest
  .spyOn(yargs as Argv, 'command')
  .mockReturnValue(yargs as Argv);
const demandCommandSpy = jest
  .spyOn(yargs as Argv, 'demandCommand')
  .mockReturnValue(yargs as Argv);
```